cpufreq/src/cpufreq-applet.c

```diff
@@ -130,563 +130,566 @@ G_DEFINE_TYPE (CPUFreqApplet, cpufreq_applet, PANEL_TYPE_APPLET)
-        unit_label = cpufreq_utils_get_frequency_unit (freq);
-
-        if (applet->show_freq) {
-                /*Force the label to render if frequencies are not found right away*/
-                if (freq_label == NULL){
-                        gtk_label_set_text (GTK_LABEL (applet->label),"---");
-                }
-                else{
-                        gtk_label_set_text (GTK_LABEL (applet->label), freq_label);
-                }
-                /*Hold the largest size set by any jumping text */
-                gtk_widget_get_preferred_size (GTK_WIDGET (applet->label),&req, NULL);
-                gtk_widget_set_size_request (GTK_WIDGET (applet->label),req.width, req.height);
-        }
-
-        if (applet->show_perc) {
-                gchar *text_perc;
-
-                text_perc = g_strdup_printf ("%d%%", perc);
-                gtk_label_set_text (GTK_LABEL (applet->label), text_perc);
-                g_free (text_perc);
-        }
-
-        if (applet->show_unit) {
-                gtk_label_set_text (GTK_LABEL (applet->unit_label), unit_label);
-                /*Hold the largest size set by MHZ or GHZ to prevent jumping */
-                gtk_widget_get_preferred_size (GTK_WIDGET (applet->unit_label),&req, NULL);
-                gtk_widget_set_size_request (GTK_WIDGET (applet->unit_label),req.width, req.height);
-        }
-
-        if (applet->show_icon) {
-                cpufreq_applet_pixmap_set_image (applet, perc);
-        }
-
-	if (governor) {
-		gchar *gov_text;
-
-		gov_text = g_strdup (governor);
-		gov_text[0] = g_ascii_toupper (gov_text[0]);
-		text_mode = g_strdup_printf ("%s\n%s %s (%d%%)",
-					     gov_text, freq_label,
-					     unit_label, perc);
-		g_free (gov_text);
-	}
-
-        g_free (freq_label);
-        g_free (unit_label);
-
-	if (text_mode) {
-		gchar *text_tip;
-
-		text_tip = cpufreq_utils_get_n_cpus () == 1 ?
-			g_strdup_printf ("%s", text_mode) :
-			g_strdup_printf ("CPU %u - %s", cpu, text_mode);
-		g_free (text_mode);
-
-		gtk_widget_set_tooltip_text (GTK_WIDGET (applet), text_tip);
-		g_free (text_tip);
-	}
-
-        /* Call refresh only the first time */
-        if (applet->need_refresh) {
-                cpufreq_applet_refresh (applet);
-		applet->need_refresh = FALSE;
+   gchar          *text_mode = NULL;
+   gchar          *freq_label, *unit_label;
+   gint            freq;
+   gint            perc;
+   guint           cpu;
+   GtkRequisition  req;
+   const gchar    *governor;
+
+   cpu = cpufreq_monitor_get_cpu (monitor);
+   freq = cpufreq_monitor_get_frequency (monitor);
+   perc = cpufreq_monitor_get_percentage (monitor);
+   governor = cpufreq_monitor_get_governor (monitor);
+
+   freq_label = cpufreq_utils_get_frequency_label (freq);
+   unit_label = cpufreq_utils_get_frequency_unit (freq);
+
+   if (applet->show_freq) {
+       /*Force the label to render if frequencies are not found right away*/
+       if (freq_label == NULL){
```

Comment:
```diff
-       if (freq_label == NULL){
+       if (freq_label == NULL) {
```

Reply:
@rbuj 
thx for looking into it.
All done, ping me when i can add the other commits.

Reply:
```patch
diff --git a/cpufreq/src/cpufreq-applet.c b/cpufreq/src/cpufreq-applet.c
index 59dde2ff..6963ed3a 100644
--- a/cpufreq/src/cpufreq-applet.c
+++ b/cpufreq/src/cpufreq-applet.c
@@ -41,33 +41,33 @@
 #include "cpufreq-utils.h"
 
 struct _CPUFreqApplet {
-    MatePanelApplet      base;
+    MatePanelApplet        base;
 
     /* Visibility */
-    CPUFreqShowMode      show_mode;
-    CPUFreqShowTextMode  show_text_mode;
-    gboolean             show_freq;
-    gboolean             show_perc;
-    gboolean             show_unit;
-    gboolean             show_icon;
-
-    CPUFreqMonitor      *monitor;
-
-    MatePanelAppletOrient orient;
-    gint                 size;
-
-    GtkWidget           *label;
-    GtkWidget           *unit_label;
-    GtkWidget           *icon;
-    GtkWidget           *box;
-    GtkWidget           *labels_box;
-    GtkWidget           *container;
-    cairo_surface_t     *surfaces[5];
-
-    gboolean             need_refresh;
-
-    CPUFreqPrefs        *prefs;
-    CPUFreqPopup        *popup;
+    CPUFreqShowMode        show_mode;
+    CPUFreqShowTextMode    show_text_mode;
+    gboolean               show_freq;
+    gboolean               show_perc;
+    gboolean               show_unit;
+    gboolean               show_icon;
+
+    CPUFreqMonitor        *monitor;
+
+    MatePanelAppletOrient  orient;
+    gint                   size;
+
+    GtkWidget             *label;
+    GtkWidget             *unit_label;
+    GtkWidget             *icon;
+    GtkWidget             *box;
+    GtkWidget             *labels_box;
+    GtkWidget             *container;
+    cairo_surface_t       *surfaces[5];
+
+    gboolean               need_refresh;
+
+    CPUFreqPrefs          *prefs;
+    CPUFreqPopup          *popup;
 };
 
 struct _CPUFreqAppletClass {
@@ -254,7 +254,8 @@ cpufreq_applet_dispose (GObject *widget)
 }
 
 static void
-cpufreq_applet_size_allocate (GtkWidget *widget, GtkAllocation *allocation)
+cpufreq_applet_size_allocate (GtkWidget     *widget,
+                              GtkAllocation *allocation)
 {
     CPUFreqApplet *applet;
     gint           size = 0;
@@ -303,14 +304,14 @@ cpufreq_applet_menu_popup (CPUFreqApplet *applet,
     /* Set up theme and transparency support */
     GtkWidget *toplevel = gtk_widget_get_toplevel (menu);
     /* Fix any failures of compiz/other wm's to communicate with gtk for transparency */
-    GdkScreen *screen = gtk_widget_get_screen (GTK_WIDGET(toplevel));
+    GdkScreen *screen = gtk_widget_get_screen (GTK_WIDGET (toplevel));
     GdkVisual *visual = gdk_screen_get_rgba_visual (screen);
     gtk_widget_set_visual (GTK_WIDGET (toplevel), visual);
     /* Set menu and it's toplevel window to follow panel theme */
     GtkStyleContext *context;
     context = gtk_widget_get_style_context (GTK_WIDGET (toplevel));
-    gtk_style_context_add_class (context,"gnome-panel-menu-bar");
-    gtk_style_context_add_class (context,"mate-panel-menu-bar");
+    gtk_style_context_add_class (context, "gnome-panel-menu-bar");
+    gtk_style_context_add_class (context, "mate-panel-menu-bar");
 
     gtk_menu_popup_at_widget (GTK_MENU (menu),
                               GTK_WIDGET (applet),
@@ -342,7 +343,8 @@ cpufreq_applet_button_press (GtkWidget *widget, GdkEventButton *event)
 }
 
 static gboolean
-cpufreq_applet_key_press (GtkWidget *widget, GdkEventKey *event)
+cpufreq_applet_key_press (GtkWidget   *widget,
+                          GdkEventKey *event)
 {
     CPUFreqApplet *applet;
 
@@ -366,7 +368,8 @@ cpufreq_applet_key_press (GtkWidget *widget, GdkEventKey *event)
 }
 
 static void
-cpufreq_applet_change_orient (MatePanelApplet *pa, MatePanelAppletOrient orient)
+cpufreq_applet_change_orient (MatePanelApplet       *pa,
+                              MatePanelAppletOrient  orient)
 {
     CPUFreqApplet *applet;
     GtkAllocation  allocation;
@@ -477,7 +480,8 @@ cpufreq_applet_about_cb (GtkAction     *action,
 }
 
 static void
-cpufreq_applet_pixmap_set_image (CPUFreqApplet *applet, gint perc)
+cpufreq_applet_pixmap_set_image (CPUFreqApplet *applet,
+                                 gint           perc)
 {
     gint image;
     gint scale;
@@ -608,7 +612,7 @@ cpufreq_applet_update_visibility (CPUFreqApplet *applet)
 }
 
 static void
-cpufreq_applet_update (CPUFreqApplet *applet,
+cpufreq_applet_update (CPUFreqApplet  *applet,
                        CPUFreqMonitor *monitor)
 {
    gchar          *text_mode = NULL;
@@ -710,13 +714,13 @@ cpufreq_applet_refresh (CPUFreqApplet *applet)
 
     /* We want a fixed label size, the biggest */
 
-    gtk_widget_get_preferred_width(GTK_WIDGET(applet->label), &label_size, NULL);
+    gtk_widget_get_preferred_width (GTK_WIDGET (applet->label), &label_size, NULL);
     total_size += label_size;
 
-    gtk_widget_get_preferred_width(GTK_WIDGET(applet->unit_label), &unit_label_size, NULL);
+    gtk_widget_get_preferred_width (GTK_WIDGET (applet->unit_label), &unit_label_size, NULL);
     total_size += unit_label_size;
 
-    gtk_widget_get_preferred_width(GTK_WIDGET(applet->icon), &pixmap_size, NULL);
+    gtk_widget_get_preferred_width (GTK_WIDGET (applet->icon), &pixmap_size, NULL);
     total_size += pixmap_size;
 
     if (applet->box) {
@@ -857,7 +861,9 @@ cpufreq_applet_setup (CPUFreqApplet *applet)
 }
 
 static gboolean
-cpufreq_applet_factory (CPUFreqApplet *applet, const gchar *iid, gpointer gdata)
+cpufreq_applet_factory (CPUFreqApplet *applet,
+                        const gchar   *iid,
+                        gpointer       gdata)
 {
     gboolean retval = FALSE;
 
```

Reply:
I think it will be ready to go with my last suggestion, keep in mind that this is a visual review ;)

Reply:
Hmm, the first changed code block is only cosmetic and your personal style.

Reply:
@rbuj 
Here we go, any think left?

Reply:
As you want, don't change the first block if you think you don't need to add two spaces to keep the variable names aligned because the MatePanelAppletOrient type is too long.

Reply:
LGTM